packages/@aws-cdk/aws-iam/lib/policy-statement.ts

```diff
@@ -229,6 +243,13 @@ export interface PolicyStatementProps {
    */
   readonly actions?: string[];
 
+  /**
+   * List of not actions to add to the statement
+   *
+   * @default - no actions
+   */
+  readonly notactions?: string[];
```

Comment:
Shouldn't this be called `notActions`? Camel case?

Reply:
You are right, it should have been.